mutmut/__main__.py

```diff
@@ -219,6 +219,13 @@ def _mutmut_trampoline(orig, mutants, *args, **kwargs):
 """
 yield_from_trampoline_impl = trampoline_impl.replace('result = ', 'result = yield from ').replace('_mutmut_trampoline', '_mutmut_yield_from_trampoline')
 
+def copy_src_dir():
```

Comment:
This entire function could be just `shutil.copytree` no?

Reply:
I thought so too but when I rewrote it it was not working as expected. I'll rework it and test it again because that should be much faster

Reply:
Ah I see why I was running into that issue.

```
def create_mutants_for_file(filename, output_path):
    input_stat = os.stat(filename)

    if output_path.exists() and output_path.stat().st_mtime == input_stat.st_mtime:
        # print('    skipped', output_path, 'already up to date')
        return
```

It was triggering this conditional and not creating the mutated files

Reply:
commenting it out works as expected now, thoughts?

Reply:
Looks more reasonable yea. If you remove the commented out code I can merge it.